store/store.go

```diff
@@ -0,0 +1,16 @@
+package store
+
+import "github.com/gopheracademy/congo"
+
+// Interface is a super interface of all substores.
+type Interface interface {
+	UserStore
+}
```

Comment:
This is an uber-interface for all stores. It's called `Interface` because the caller will see it as `store.Interface`. If we called it `Store` then the caller would see it as `store.Store` which sounds kinda dumb. :)